src/Microsoft.AspNetCore.ResponseCaching/Internal/StringBuilderExtensions.cs

```diff
@@ -10,10 +9,13 @@ internal static class StringBuilderExtensions
     {
         internal static StringBuilder AppendUpperInvariant(this StringBuilder builder, string value)
         {
-            builder.EnsureCapacity(builder.Length + value.Length);
-            for (var i = 0; i < value.Length; i++)
+            if (string.IsNullOrEmpty(value))
```

Comment:
Should this be inverted? NOT string.IsNullOrEmpty?

Reply:
Yup, bad typo during fixup.

Reply:
Pushed fix, ran the tests again to make sure.

Reply:
Thanks!